src/clients/java/samples/basic/pom.xml

```diff
@@ -39,8 +39,7 @@
     <dependency>
       <groupId>com.tigerbeetle</groupId>
       <artifactId>tigerbeetle-java</artifactId>
-      <!-- Grab the latest commit from: https://repo1.maven.org/maven2/com/tigerbeetle/tigerbeetle-java/maven-metadata.xml -->
-      <version>0.0.1-3431</version>
+      <version>0.0.0</version>
```

Comment:
I wasn't able to find a clean way to use "path dependencies" with maven, so this is just a placeholder, and some extra zig code is needed so that maven finds the correct code. 

Still, we avoid mentioning a random version.

Reply:
Perhaps a comment saying that?